src/constants/common.js

```diff
@@ -1 +1,8 @@
 export const FACEBOOK_PERMISSIONS = ['email'];
+export const INITIAL_COORDS = {
```

Comment:
I think you should'nt have initial coords. Maybe try to have them before showing the map?